lib/ModelMixins/ClippingMixin.ts

```diff
@@ -200,50 +200,54 @@ function ClippingMixin<T extends Constructor<BaseType>>(
         this._clippingBoxDrawing.setTransform(boxTransform);
         this._clippingBoxDrawing.keepBoxAboveGround = this.clippingBox.keepBoxAboveGround;
       } else {
-        this._clippingBoxDrawing = new BoxDrawing(cesium, boxTransform, {
-          keepBoxAboveGround: this.clippingBox.keepBoxAboveGround,
-          onChange: action(({ modelMatrix, isFinished }) => {
-            Matrix4.multiply(
-              this.inverseClippingPlanesOriginMatrix,
-              modelMatrix,
-              this.clippingPlaneModelMatrix
-            );
-            if (isFinished) {
-              const position = Matrix4.getTranslation(
+        this._clippingBoxDrawing = BoxDrawing.fromTransform(
```

Comment:
The real change here is replacing `new BoxDrawing` with `BoxDrawing.fromTransform`, rest is indentation change.